examples/imagefilter/node/tsconfig.json

```
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
```

Let's not transpire the JS to commonjs, when nodejs can import ES modules and this way we can use top-level await.

```suggestion
        "module": "node16",
```